java-sdk/src/main/java/com/adobe/target/edge/client/http/DefaultTargetHttpClient.java

```diff
@@ -83,4 +91,7 @@ public void close() {
         unirestInstance.shutDown();
     }
 
+    UnirestInstance getUnirestInstance() {
```

Comment:
Although it has default excess, I'm not exactly sure about this. @artur-ciocanu any comments? 
This also tightly couples our tests to unirest, which is not our long term plan.

Reply:
Hi @anshchauhan , I can easily revert it back to a private method. However, it may impact our ability to have a full unit test coverage for the proxy functionality. Let me know what you think.

Reply:
Resolving this conversation as @anshchauhan has approved the changes.